rainier-core/src/main/scala/com/stripe/rainier/core/Continuous.scala

```diff
@@ -66,7 +64,7 @@ trait LocationScaleFamily { self =>
   */
 object Normal extends LocationScaleFamily {
   def logDensity(x: Real): Real =
-    (x * x) / -2.0
+    ((x * x) / -2.0) - 0.5 * Real(2 * math.Pi).log
```

Comment:
ooh I can see how this might matter in a mixture

Reply:
When I spotted that I thought for about half a minute that the whole of Rainier was out by a constant factor...